pylint/pyreverse/diagrams.py

```diff
@@ -232,29 +268,29 @@ def get_module(self, name, node):
                 return mod
         raise KeyError(name)
 
-    def add_from_depend(self, node, from_module):
+    def add_from_depend(self, node: ImportFrom, from_module: str) -> None:
         """Add dependencies created by from-imports."""
         mod_name = node.root().name
         obj = self.module(mod_name)
         if from_module not in obj.node.depends:
             obj.node.depends.append(from_module)
 
-    def extract_relationships(self):
+    def extract_relationships(self) -> None:
         """Extract relationships between nodes in the diagram."""
         super().extract_relationships()
-        for obj in self.classes():
+        for class_obj in self.classes():
             # ownership
             try:
-                mod = self.object_from_node(obj.node.root())
-                self.add_relationship(obj, mod, "ownership")
+                mod = self.object_from_node(class_obj.node.root())
+                self.add_relationship(class_obj, mod, "ownership")
```

Comment:
Those two lines were not covered before.
Funnily enough, this is (at least part) of what would be needed for #6543, but currently no `Printer` knows how to handle "ownership" relationships anyway.